src/lib/features/project/project-store.ts

```diff
@@ -140,8 +140,8 @@ class ProjectStore implements IProjectStore {
             }
         }
 
-        if (query) {
-            projects = projects.where(query);
+        if (query?.id) {
```

Comment:
the old query was only for id and I prefer to have explicit params since previously it was also catching archived boolean which is not a valid column name